Doc/library/asyncio-eventloop.rst

```diff
@@ -186,19 +186,25 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`~concurrent.futures.ThreadPoolExecutor`.
+   After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time the executor will
-   be given to finish joining. The default value is ``None``, which means the
-   executor will be given an unlimited amount of time.
+   The *timeout* parameter specifies the duration in seconds (as a :class:`float`)
+   this method gives the executor to finish joining.
+   With the default, ``None``, the
+   method allows the executor an unlimited amount of time.
```

Comment:
```suggestion
   With the default, ``None``,
   the executor is allowed an unlimited amount of time.
```

Likewise, here, I feel inserting "the method" here makes the sentence structure more complex for the reader to parse without really aiding meaning. Here, the exact implementation detail doing the allowing (the subject) is a lot less important than the object being allowed (the executor), which is exactly what passive voice is useful for. And the method is explicitly what we are describing here, which the basic structure of API reference documentation already makes clear, and repeatedly restating is not really in line with the concision and directness that serves reference-type docs well.

Reply:
or 'the executor has ....

Reply:
Like the above, that framing seems more appropriate for an intrinsic requirement than an externally-set  discretionary deadline. But if we do change the above, we should presumably change this to match